rotkehlchen/chain/ethereum/modules/yearn/vaults.py

```diff
@@ -470,6 +470,9 @@ def _get_vault_deposit_events(
             )
             tx_hash = deserialize_evm_tx_hash(deposit_event['transactionHash'])
             tx_receipt = self.ethereum.get_transaction_receipt(tx_hash)
+            if tx_receipt is None:
```

Comment:
This is an error.
```suggestion
            if tx_receipt is None:
                log.error(f'Yearn deposit event has transaction {tx_hash.hex()} but it could not be found when querying a node')
```